### PR TITLE
Skip meshmap video download/render for mobile view

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

**Description**
Avoiding the download of meshmap video in the banner for mobile view by limiting the rendering based on the `window.innerWidth` property.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
Seems like it was getting downloaded even for mobile view:
Screenshot from 2022-12-05 22-57-14

File: src/sections/Home/Banner-4/index.js [hunk 1]
```diff
@@ -62,13 +62,15 @@ const Banner1 = () => {
...
             </Container>
           </Col>
-          <Col sm={4} lg={6} className="section-title-wrapper video-col">
-            <Link to="/cloud-native-management/meshmap">
-              <video autoPlay muted loop preload="metadata" className="meshmapVideo">
-                <source src={meshmapVideo} type="video/mp4"></source>
-              </video>
-            </Link>
-          </Col>
+          {typeof window != "undefined" && window.innerWidth > 760 && (
```
Comment (leecalcote) [inline src/sections/Home/Banner-4/index.js:65]: User-agent just wasn't good enough for you, huh? Getting fancy with "typeof" instead. 

All kidding aside: I suppose the combination of these properties is a broader guarantee than trying to match all-known mobile user-agents.

Comment (Nikhil-Ladha) [reply src/sections/Home/Banner-4/index.js:65]: Hehe, I had to 😎

**Comment by Nikhil-Ladha** [comment]
Looks good in the resources downloaded for mobile view.

**Pull request merged**